spec/features/initial_decision_spec.rb

```diff
@@ -24,8 +24,8 @@
 include RichTextEditorHelpers
 
 feature 'Initial Decision', js: true, sidekiq: :inline! do
-  given(:admin) { FactoryGirl.create(:user, :site_admin) }
-  given(:paper) do
```

Comment:
I had no idea

Reply:
Me neither. It's supposed to be equivalent to `let` but it seemed to fix an issue.